src/test/OrgTests/OrganizationJWTTests.java

```diff
@@ -143,4 +145,120 @@ public void organizationInviteInvalid() {
     assert (actualResponseJSON.has("status"));
     assertThat(actualResponseJSON.getString("status")).isEqualTo("EMPTY_FIELD");
   }
+
+  @Test
+  public void adminDashboardTest() {
+    TestUtils.login("adminBSM", "adminBSM");
+    HttpResponse actualResponse =
+        Unirest.post(TestUtils.getServerUrl() + "/dashboard")
+            .header("Accept", "*/*")
+            .header("Content-Type", "text/plain")
+            .asString();
+
+    JSONObject resp = TestUtils.responseStringToJSON(actualResponse.getBody().toString());
+
+    TestUtils.logout();
+
+    assert (resp.has("message"));
+    assertThat(resp.getString("message")).isEqualTo("Success.");
+    assert (resp.has("status"));
+    assertThat(resp.getString("status")).isEqualTo("SUCCESS");
+    assert (resp.has("workerCount"));
+    assertThat(resp.getInt("workerCount")).isEqualTo(7);
+    assert (resp.has("clientCount"));
+    assertThat(resp.getInt("clientCount")).isEqualTo(0);
+  }
+
+  @Test
+  public void findAllOrgs() throws ValidationException {
+    HttpResponse actualResponse =
+        Unirest.post(TestUtils.getServerUrl() + "/all-orgs")
+            .header("Accept", "*/*")
+            .header("Content-Type", "text/plain")
+            .asString();
+
+    JSONObject resp = TestUtils.responseStringToJSON(actualResponse.getBody().toString());
+
```

Comment:
![image](https://user-images.githubusercontent.com/13875633/90209412-61ef9180-ddb9-11ea-941b-84ddf4a20473.png)
for some reason your tests are failing, so i would look into that